consensus/replay_test.go

```diff
@@ -1042,7 +1036,7 @@ func (app *badApp) Commit() abci.ResponseCommit {
 //--------------------------
 // utils for making blocks
 
-func makeBlockchainFromWAL(wal WAL) ([]*types.Block, []*types.Commit, error) {
```

Comment:
@liamsi, need your feedback on this. Previously, it was possible to make blockchain from WAL for testing from msgs only, but now that requires state as well. Confirm if the solution is correct

Reply:
It's not